### DISCUSSION TITLE
article/secrets-from-the-new-science-of-expertise

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#46] [Announcements]
# article/secrets-from-the-new-science-of-expertise

著名心理学家艾利克森在“专业特长科学”领域潜心几十年，研究了一系列行业或领域中的专家级人物后发现,不论在什么行业或领域，提高技能与能力的最有效方法全都遵循一系列普遍原则，他将这种通用方法命名为“刻意练习”。对于在任何行业或领域中希望提升自己的每个人，刻意练习是黄金标准，是迄今为止发现的最强大的学习方法。

https://tangly1024.com/article/secrets-from-the-new-science-of-expertise

**Comment by lanhuakai** [comment]
想问下，您这个网站是怎么优化的，打开速度很好。是vercel的吗？还是别的

**Reply by tangly1024** [reply]
基于Vercel部署的，用CloudFlare做了CDN缓存
